docusaurus.config.js

```diff
@@ -112,6 +112,7 @@ const config = {
     {
       image: "img/restate-og.png",
       navbar: {
+        hideOnScroll: true,
```

Comment:
Right now, the problem of anchors being under the navbar has been solved by https://github.com/restatedev/documentation/pull/539. If we decide to hide the navbar on scrolling, then this change is no longer needed.